android/src/main/java/com/mapbox/mapboxgl/GlobalMethodHandler.java

```diff
@@ -65,6 +65,10 @@ public void onMethodCall(MethodCall methodCall, MethodChannel.Result result) {
         OfflineManagerUtils.setOfflineTileCountLimit(
             result, context, methodCall.<Number>argument("limit").longValue());
         break;
+      case "setHttpHeaders":
+        Map<String, String> headers = (Map<String, String>) methodCall.argument("headers");
+        MapboxHttpRequestUtil.setHttpHeaders(headers);
```

Comment:
set the result based on setHttpHeaders - might even be a good idea to pass it to setHttpHeaders and set it in there

Reply:
I hope I understood correctly.